lib/msal-core/package.json

```diff
@@ -28,50 +28,60 @@
     "clean": "shx rm -rf dist docs lib-commonjs lib-es6",
     "doc": "typedoc --out ./docs ./src/ --gitRevision dev",
     "build:modules": "tsc && tsc -m es6 --outDir lib-es6",
-    "build": "npm run clean && npm run doc && npm run build:modules && npm run lint && npm run build:webpack && npm test",
-    "test": "karma start && jasmine tests/MSALSpec.js",
+    "build": "npm run clean && npm run doc && npm run build:modules && npm run lint && npm run build:webpack && npm run test",
```

Comment:
no need to do npm run test, doesn't npm test just work

Reply:
npm test works. Not sure why run is added. May be just a typo.

Reply:
I have taken out 'run' in my latest commit.